### PR TITLE
jquery.a11y tabindex fix

### PR DESCRIPTION
**Pull request by tomgreenfield**
Remove tabindex from hideable elements on popdown

### CODE REVIEW

**Comment by oliverfoster** [comment]
+1

File: src/core/js/libraries/jquery.a11y.js [hunk 1]
```diff
@@ -1147,10 +1147,14 @@
...
-                    //show element to screen reader
-                    $item.removeAttr('aria-hidden').removeClass("aria-hidden");
+                    return $item.attr('aria-hidden', true).addClass("aria-hidden");
+                }
+
+                //show element to screen reader
+                $item.removeAttr('aria-hidden').removeClass("aria-hidden");
+                
+                if ($item.is(domSelectors.hideableElements)) {
+                    $item.removeAttr("tabindex");
```
Comment (oliverfoster) [inline src/core/js/libraries/jquery.a11y.js]: CODE NOTES

```.a11y-hideable```
Some elements, which were never explicitly in the tab order, need to be removed from the tab order when a popup occurs. Label tags associated with input tags are an example of such elements. When the popdown occurs their tabindex="0" should be removed as it wouldn't have originally existed and will cause a double tab if left.

iFrame containers for youtube videos will also have this same problem.

**Comment by moloko** [comment]
+1

**Comment by brian-learningpool** [comment]
+1

**Pull request merged**